packages/components/psammead-navigation/src/index.jsx

```diff
@@ -214,7 +219,8 @@ NavigationLi.defaultProps = {
 
 const StyledNav = styled.nav`
   position: relative;
-  background-color: ${C_POSTBOX};
+  ${({ isOpen }) =>
+    isOpen ? `background-color: ${C_EBON};` : `background-color: ${C_POSTBOX};`}
```

Comment:
Only the colors value is changing. May be we could do.
```js
background-color:   isOpen ? `${C_EBON}` : `${C_POSTBOX}`}
```

Reply:
Done.